widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/SecureConversationsRepository.kt

```diff
@@ -69,11 +69,31 @@ internal class SecureConversationsRepository(private val core: GliaCore, private
 
     fun getUnreadMessagesCount(callback: RequestCallback<Int>) = secureConversations.getUnreadMessageCount(callback)
 
-    fun getHasPendingSecureConversations(): Single<Boolean> = Single.create { emitter ->
-        secureConversations.getPendingSecureConversationStatus { hasPendingSecureConversations, _ ->
-            if (!emitter.isDisposed) {
-                emitter.onSuccess(hasPendingSecureConversations ?: false)
+    fun subscribeToUnreadMessagesCount(callback: RequestCallback<Int>) = secureConversations.subscribeToUnreadMessageCount(callback)
+
+    fun unsubscribeFromUnreadMessagesCount(callback: RequestCallback<Int>) = secureConversations.unSubscribeFromUnreadMessageCount(callback)
+
+    fun unreadMessagesCountObservable(): Observable<Int> = Observable.create { emitter ->
```

Comment:
NIT. I prefer a getUnreadMessagesCountObservable() name for a function. Let me know if you have points against this.

Reply:
The same as above.

Reply:
changed